CourseManager/Views/ExampleViews/ProgressView/CircleProgressView.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  CircleProgressView.swift
+//  CourseManager
+//
+//  Created by Summer Crow on 2022-12-14.
+//
+
+import SwiftUI
+
+struct CircleProgressView: View {
+    var value: CGFloat = 0.95
```

Comment:
@sghourab variables `value` and `percent` should be sync which each other. `percent` should be readonly variable and when we set `value` it should be updated.